arbnode/batch_poster.go

```diff
@@ -1935,6 +1921,17 @@ func (b *BatchPoster) MaybePostSequencerBatch(ctx context.Context) (bool, error)
 		return false, fmt.Errorf("decoding batch position: %w", err)
 	}
 
+	if b.config().IsDecentralizedTimeboost {
+		// TODO: Fallback if leader fails to post batch
+		leader, err := b.batchVerifier.IsLeaderForBatch(batchPosition.NextSeqNum)
+		if err != nil {
```

Comment:
I feel like if there is an error, it should just default to post the batch right?